modules/CustomButton/resources/customButton.js

```diff
@@ -3,6 +3,7 @@
 			//indicates we were explicitly asked to show the button (will be used when re-enabling the button)
 			shouldShow : false,
 			isDisabled: false,
+			ariaText: '',
```

Comment:
I'd add a default here from the plugin localization texts, maybe the same as the label?